### PR TITLE
Plural compact to unsupported

### PR DESCRIPTION
**Pull request by sven-oly**
Reclassify failures in handling compact numbers in plural rules as "unsupported".

Note that there are still lots of test failures on each platform.

### CODE REVIEW

File: executors/cpp/plural_rules.cpp [hunk 1]
```diff
@@ -53,21 +56,40 @@ string TestPluralRules (json_object* json_in) {
...
+          "error_detail",
+          json_object_new_string(sample_string.c_str()));
+      return json_object_to_json_string(return_json);
     }
 
     if (sample_string.find('.') != std::string::npos) {
       // Convert into an integer, decimal, or compact decimal
       input_double_sample = std::stod(sample_string);
       input_is_double = true;
+      cout << "Double sample: " << input_double_sample << endl;
```
Comment (echeran) [inline executors/cpp/plural_rules.cpp]: Remove stdout print debugging (& related `#include`s and namespace `using`s).

**Review by echeran — COMMENTED** [review]
Looks good apart from stdout debugging lines

**Review by echeran — APPROVED** [review]
LGTM

**Comment by sven-oly** [comment]
#339

**Pull request merged**